packages/apm/src/integrations/tracing.ts

```diff
@@ -565,6 +620,25 @@ export class Tracing implements Integration {
     return time / 1000;
   }
 
+  /**
+   * Adds debug data to the span
+   */
+  private static _addSpanDebugInfo(span: Span): void {
+    if (global.performance) {
+      span.setData('performance', true);
+      span.setData('performance.timeOrigin', global.performance.timeOrigin);
+      span.setData('performance.now', global.performance.now());
+      // tslint:disable-next-line: deprecation
+      if (global.performance.timing && global.performance.timing.navigationStart) {
+        // tslint:disable-next-line: deprecation
+        span.setData('performance.timing.navigationStart', performance.timing.navigationStart);
+      }
+    } else {
+      span.setData('performance', false);
+    }
+    span.setData('Date.now()', Date.now());
+  }
```

Comment:
Suggestion:

Add all of this debug info to a single object in `data`, will make it easier on us later if there is a namespace (instead mixing everything in the top level of `data`).

e.g.

```
{
data: {
  __sentry_debug: {
    performance: true ...
```